packages/hyperion-autologging/src/ALInteractableDOMElement.ts

```diff
@@ -148,7 +148,7 @@ let ignoreInteractiveElement: (node: Element) => boolean = node => {
    * Later, when we wanted to remove the flag, the cleanup is easy.
    */
   function ignoreInteractiveElementCore(node: Element): boolean {
-    return node.tagName === 'BODY' || node.tagName === 'HTML' ||
+    return node.tagName === 'BODY' || node.tagName === 'HTML' || node.tagName === 'LABEL' ||
```

Comment:
This will be very hard to remember later. Please add a comment like this:

According to mdn, a click on label, creates a click on the input. https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/label
We don't want to double count the click, and want to ignore the one on the lable. By not marking it interactive, we would prevent `captureHandler` to run. 

So, that means our problem. is